resources/views/pages/user-shifts.html

```diff
@@ -52,6 +52,9 @@ <h1>%title% %add_link%</h1>
                 <div class="btn-group mb-1" role="group">
                     %dashboard%
                 </div>
+                <div class="btn-group mb-1" role="group">
```

Comment:
We now have two public dashboard buttons 😄 Could you reduce it to one please.

Reply:
whoops, that wasn't on purpose :D will do!